.env.example

```diff
@@ -18,9 +18,14 @@ BEE_ENDPOINT=
 CHAIN_ID=7011893055
 
 GOOGLE_STORAGE_BUCKET_NAME=blobscan-test-bucket
-GOOGLE_STORAGE_PROJECT_ID=blobscan-379218
+GOOGLE_STORAGE_PROJECT_ID=blobscan-test-project
+GOOGLE_STORAGE_API_ENDPOINT=http://localhost:4443
```

Comment:
Let's leave this as `GOOGLE_STORAGE_API_ENDPOINT=` since this variable is only used in development and it is currently hardcoded in the docker-compose.yml file